docs/changelog-fragments/620.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed crash when downloading non-existent remote file using SCP -- by :user:`Jakuje`
```

Comment:
This would work. I think that it could also be something like
```suggestion
Downloading non-existent remote files via SCP no longer crashes the program -- by :user:`Jakuje`
```

Reply:
Updated as suggested.